test/e2e/storage/testsuites/volumemode.go

```diff
@@ -426,3 +427,32 @@ func swapVolumeMode(podTemplate *v1.Pod) *v1.Pod {
 	}
 	return pod
 }
+
+// NOTE(avalluri): The below code is intentionally copied from e2e/common package.
```

Comment:
Can we not put this in a test framework utility package and import that instead?

Reply:
I've opened for https://github.com/kubernetes/kubernetes/issues/83937 for that. In the meantime, can we backport this less intrusive solution?